android/src/main/java/com/blueshift/reactnative/BlueshiftReactNativeModule.java

```diff
@@ -221,11 +221,9 @@ void displayInAppNotification() {
                 .displayInAppMessages();
     }
 
-    @ReactMethod
-    void registerForInAppMessage() {
```

Comment:
Will this impact any of our existing customer?

Reply:
If they update to this new version, they might see a build error if they are not passing any screen name.